modules/claims_api/app/workers/claims_api/claim_uploader.rb

```diff
@@ -24,12 +25,37 @@ def perform(uuid)
         uploader.retrieve_from_store!(claim_object.file_data['filename'])
         file_body = uploader.read
         ClaimsApi::Logger.log('526', claim_id: auto_claim.id, attachment_id: uuid)
-        service(auth_headers).upload(file_body, claim_upload_document(claim_object))
+        if Flipper.enabled? :claims_claim_uploader_use_bd
+          claim_bd_upload_document(auto_claim, uploader&.file&.file)
+        else
+          EVSS::DocumentsService.new(auth_headers).upload(file_body, claim_upload_document(claim_object))
+        end
       end
     end
 
     private
 
+    def claim_bd_upload_document(claim, pdf_path)
+      ClaimsApi::BD.new.upload(claim:, pdf_path:)
+    # Temporary errors (returning HTML, connection timeout), retry call
+    rescue Faraday::Error::ParsingError, Faraday::TimeoutError => e
+      ClaimsApi::Logger.log('benefits_documents',
+                            retry: true,
+                            detail: "/upload failure for claimId #{claim&.id}: #{e.message}")
+      raise e
+    # Permanent failures, don't retry
+    rescue => e
+      message = if e.respond_to? :original_body
+                  e.original_body
+                else
+                  e.message
+                end
+      ClaimsApi::Logger.log('benefits_documents',
+                            retry: false,
+                            detail: "/upload failure for claimId #{claim&.id}: #{message}")
```

Comment:
Is there a chance the `message` would have PII?

Reply:
I don't believe so, not from what they return, at least _they_ shouldn't be returning PII either.... so I think we're safe to log this. let's monitor when we turn it on though, just to see if we do have concern with any of the details being returned.